src/biogeochem/CNNStateUpdate1Mod.F90

```diff
@@ -43,12 +46,13 @@ subroutine NStateUpdateDynPatch(bounds, num_soilc_with_inactive, filter_soilc_wi
     type(cnveg_nitrogenflux_type)           , intent(in)    :: cnveg_nitrogenflux_inst
     type(cnveg_nitrogenstate_type)          , intent(inout) :: cnveg_nitrogenstate_inst
     type(soilbiogeochem_nitrogenstate_type) , intent(inout) :: soilbiogeochem_nitrogenstate_inst
+    type(soilbiogeochem_nitrogenflux_type) , intent(inout) :: soilbiogeochem_nitrogenflux_inst
     !
     ! !LOCAL VARIABLES:
     integer  :: c   ! column index
     integer  :: g   ! gridcell index
     integer  :: fc  ! column filter index
-    integer  :: j   ! level index
+    integer  :: j, nstep ! level index
```

Comment:
In nstep level index?! Please stay consistent with the code. Probably nstep is added here by mistake!